tmt/cli/_root.py

```diff
@@ -337,6 +337,11 @@ def main(
          What to do when plan fails to finish. Quit by default, or continue with the next plan.
          """,
 )
+@option(
+    '--recipe',
+    is_flag=True,
+    help='Generate a recipe file at the end of the run.',
+)
```

Comment:
I thought we need to reserve this flag for the user to pass their own recipe. As for generating the recipe, it would be better to make it unconditional right now.